recipes/larray-editor/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+"$PYTHON" setup.py install || exit 1
```

Comment:
In https://github.com/ContinuumIO/menuinst/wiki/Menus-in-Conda-Recipes there are instructions for menus on Linux & MacOs. Did you try those?

Reply:
I'll try again as soon as possible.